### ISSUE TITLE
FontForge crashes when clicking "OK" in the Font Info

### ISSUE THREAD

**Issue by Catterio** [fontforge/fontforge#5231]
* Provide a general summary of the issue in the **Title** above.  
* Before you open an issue, please check if a similar issue already exists or has been closed before.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### When reporting a bug/issue:
- [ ] Screenshot
- [ ] The FontForge version and the operating system you're using
- [ ] The behavior you expect to see, and the actual behavior
- [ ] Steps to reproduce the behavior 
- [ ] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [ ] A description of the problem you're trying to solve, including _why_ you think this is a problem
- [ ] If the feature changes current behavior, reasons why your solution is better
- [ ] \(optional) Possible solution/fix/workaround

**Comment by Catterio** [comment]
One day, while I was trying to make an italic style of a font, I tried to edit the Font Info of the font and when I was done, FontForge immediately crashes the second I clicked OK to confirm the changes. I tested this again in another new font _(I did no changes to this one)_ and FontForge also crashes when I click OK in the Font Info. One time, when I try reinstalling the program and restarting my machine over and over again, FontForge somehow didn't crash. But now when I got this exact same problem again, I seriously need a solution to this otherwise it'd be impossible for me to edit the font names. Although I don't know if it's only me that experiences this issue but the bug is very annoying. The FontForge version that I used is the 2023-01-01 release.

Clip of me replicating the process is through the link below:
https://user-images.githubusercontent.com/120480323/235137750-4bd4eeef-968b-4529-8435-f93413ed6e13.mp4

I hope that there will soon be a solution to solve this issue so that I can continue my work again as all my works have been continuously halted by this bug.

**Comment by jtanx** [comment]
do you have a sample font

**Comment by Catterio** [comment]
> do you have a sample font
Yes, I've also tried others but it seems that this issue only occurs with this font file below, other font files seem to work fine. You can extract the font file in the ZIP and try replicating the issue on it.

[BreezeSans-Bold.zip](https://github.com/fontforge/fontforge/files/11358498/BreezeSans-Bold.zip)

**Comment by jtanx** [comment]
The gsub table in that font is kind of broken, specifically this:

> ERROR: Layout: Bad lookup list index 64 in lookup record

FF is actually validating this but then it ends up partially loading the lookups which then causes it to crash when it tries to use this down the line

**Comment by Catterio** [comment]
> The gsub table in that font is kind of broken, specifically this:
> 
> ERROR: Layout: Bad lookup list index 64 in lookup record
> 
> FF is actually validating this but then it ends up partially loading the lookups which then causes it to crash when it tries to use this down the line
Thanks for the reply, but is there a way to fix this?
